src/index.css

```diff
@@ -1,20 +1,31 @@
+:root {
+  --dark-gray: #858694;
+  --light-gray: #e0e0e0;
+  --orange: #f5913e;
+  --border: #bababa;
+}
+
 #root {
-  width: 100%;
+  width: 700px;
   height: 100%;
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
 }
 
 .display-container {
```

Comment:
- [ ] [REQUIRED] result should be aligned to the right ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-alignment.png))
- [ ] [REQUIRED] result should be presented with white and bold text ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/display-text.png))

Reply:
Result has been aligned to the right and presented with white and bold text. Also, font sizes has been changed.